### ISSUE TITLE
Шифровка всего трафика

### ISSUE THREAD

**Issue by RomgoTM** [Flowseal/zapret-discord-youtube#614]
Здравствуйте, есть ли возможность шифровать не только выбранные домены сайтов, а весь трафик в целом?

**Comment by serpen7** [comment]
эм а это разве прога для шифрования, а не дурения ТСПУ? Если вы считаете это ВПНом то тут вы ошибаетесь. Оно не шифрует трафик.

**Comment by RomgoTM** [comment]
Нет, я не считаю данную программу ВПНом. Я прекрасно знаю разницу 
Поставлю вопрос по другому: 
Можно ли заставить программу дурить весть трафик/пакеты данных, на подобии ByeDPI?

**Comment by serpen7** [comment]
можно, но это черевато поломками или непониманием со стороны сервисов которые вы пользуете, поэтому так же как в гудбайдпи тут листы, чтобы не ломать если это случится, все. Т.к. к этим ресурсам и так нет доступа. А еще это будет хорошо так нагружать процессор. Просто не могу понять зачем вам это нужно т.к. вам же не заблокировали все вообще, так зачем вся эта манипуляция будет нужна на нормальном трафике создавая кучу всего того что ненужно во всем трафике?

**Comment by Eptiy** [comment]
Если нужно разблокировать больше сайтов, чем предлагается этим билдом, можно использовать лист заблокированных доменов отсюда: https://github.com/1andrevich/Re-filter-lists.

А так, да, смысла весь трафик гонять в таком мудрёном виде нет.

**Comment by 0ldir** [comment]
> Здравствуйте, есть ли возможность шифровать не только выбранные домены сайтов, а весь трафик в целом?

у меня получилось, я поставил "*" в list-general.txt и все заработало
